### ISSUE TITLE
v3.7.0 Node Sass does not yet support your current environment: Linux 64-bit with false

### ISSUE THREAD

**Issue by xlegs** [sass/node-sass#1537]
Installed via npm on Manjaro Linux. My gulp.js build now encounters an error.

```
[13:12:45] Starting 'sass'...
[13:12:45] 'sass' errored after 198 ms
[13:12:45] Error: Node Sass does not yet support your current environment: Linux 64-bit with false
For more information on which environments are supported please see:
TODO URL
    at Object.<anonymous> (XXX/node_modules/node-sass/lib/index.js:12:11)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
    at Function.Module._load (module.js:407:3)
    at Module.require (module.js:466:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (XXX/node_modules/gulp-sass/index.js:187:21)
    at Module._compile (module.js:541:32)
```

**Comment by xzyfer** [comment]
npm update

In future please search old issues before opening new ones. This has been
reported many times.

**Comment by xlegs** [comment]
Didn't work. npm packages are up-to-date.

**Comment by xzyfer** [comment]
This message was fixed in v3.7.0 so if you're seeing it you must not be on the latest version.

Please provide the output of the following commands

```
npm -v              
node -v
node -p process.versions
node -p process.platform
node -p process.arch
npm ls node-sass
```

**Comment by xlegs** [comment]
'npm ls node-sass' showed

'''
└─┬ gulp-sass@2.3.1
  └── node-sass@3.6.0 
'''

After removing and reinstalling gulp-sass (as per [these instructions](https://github.com/dlmanning/gulp-sass/wiki/Update-to-the-latest-Gulp-Sass)), my node-sass was updated and all was well.

Thanks.
